src/m365/spo/commands/listitem/listitem-roleinheritance-reset.spec.ts

```diff
@@ -0,0 +1,197 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import appInsights from '../../../../appInsights';
+import auth from '../../../../Auth';
+import { Logger } from '../../../../cli';
+import Command, { CommandError } from '../../../../Command';
+import request from '../../../../request';
+import { sinonUtil } from '../../../../utils';
+import commands from '../../commands';
+const command: Command = require('./listitem-roleinheritance-reset');
+
+describe(commands.LISTITEM_ROLEINHERITANCE_RESET, () => {
+  let log: any[];
+  let logger: Logger;
+
+  before(() => {
+    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
+    sinon.stub(appInsights, 'trackEvent').callsFake(() => { });
+    auth.service.connected = true;
+  });
+
+  beforeEach(() => {
+    log = [];
+    logger = {
+      log: (msg: string) => {
+        log.push(msg);
+      },
+      logRaw: (msg: string) => {
+        log.push(msg);
+      },
+      logToStderr: (msg: string) => {
+        log.push(msg);
+      }
+    };
+  });
+
+  afterEach(() => {
+    sinonUtil.restore([
+      request.post
+    ]);
+  });
+
+  after(() => {
+    sinonUtil.restore([
+      auth.restoreAuth,
+      appInsights.trackEvent
+    ]);
+    auth.service.connected = false;
+  });
+
+  it('has correct name', () => {
+    assert.strictEqual(command.name.startsWith(commands.LISTITEM_ROLEINHERITANCE_RESET), true);
+  });
+
+  it('has a description', () => {
+    assert.notStrictEqual(command.description, null);
+  });
+
+  it('supports debug mode', () => {
+    const options = command.options();
+    let containsDebugOption = false;
+    options.forEach(o => {
+      if (o.option === '--debug') {
+        containsDebugOption = true;
+      }
+    });
+    assert(containsDebugOption);
+  });
+
+  it('supports specifying URL', () => {
+    const options = command.options();
+    let containsTypeOption = false;
+    options.forEach(o => {
+      if (o.option.indexOf('<webUrl>') > -1) {
+        containsTypeOption = true;
+      }
+    });
+    assert(containsTypeOption);
+  });
+
+  it('defines correct option sets', () => {
+    const optionSets = command.optionSets();
+    assert.deepStrictEqual(optionSets, [['listId', 'listTitle']]);
+  });
+
+  it('fails validation if the webUrl option is not a valid SharePoint site URL', (done) => {
+    const actual = command.validate({ options: { webUrl: 'foo', id: 8, listTitle: 'Demo List' } });
+    assert.notStrictEqual(actual, true);
+    done();
+  });
+
+  it('passes validation if the webUrl option is a valid SharePoint site URL', (done) => {
+    const actual = command.validate({ options: { webUrl: 'https://contoso.sharepoint.com', id: 8, listId: '0cd891ef-afce-4e55-b836-fce03286cccf' } });
+    assert(actual);
```

Comment:
I would change this to `assert.strictEqual(actual, true);`  I think now the test may pass even if we pass invalid `listId` 🤔

Reply:
✅